runtime/compiler/p/runtime/CMakeLists.txt

```diff
@@ -58,7 +53,5 @@ j9jit_files(
 	p/runtime/Recomp.cpp
 	p/runtime/Recompilation.spp
 	p/runtime/PPCHWProfiler.cpp
-	p/runtime/PPCLMGuardedStorage.cpp
-	p/runtime/ebb.spp
```

Comment:
ebb is the general mechanism for user-level interrupt handling, used currently by both HWProfiling and LMGuardedStorage. I figured you cannot remove ebb.spp file from the build, unless you remove RI (RuntimeInstrumentation) too.

Reply:
Oops, looks like I accidentally removed an extra line here without noticing :sweat_smile:. Should be fixed now.